### ISSUE TITLE
[use-cases] Grammar 5.5.3 Example 5.3

### ISSUE THREAD

**Issue by lauracarlson** [w3c/silver#696]
The [Use Cases for WCAG 3.0 Conformance](https://github.com/w3c/silver/blob/main/use-cases/index.html) document has a couple of grammar issue in [5.5.3 Example 5.3](https://w3c.github.io/silver/use-cases/#example-5-3):

"The company is considering to switch tools"
should likely be
"The company is considering switching tools"

and 

"in an accessibility statements"
should be
"in an accessibility statement"

**Comment by cwadamsoforacle** [comment]
This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
